content/_index.en.md

```diff
@@ -277,14 +277,22 @@ GITHUB_TOKEN=ghp_XXXXXXXXX npx ota dataset --schedule --publish --remove-local-c
 npx ota serve
 ```
 
-Start the collection metadata API server. 
+Start the collection metadata API server.
 
 The API will be available under `http://localhost:<port>/<basePath>/<apiVersion>/<resource>`.
 The server `<port>` and `<basePath>` are defined in the [configuration](#configuring).
 
 For example, with the default configuration, the list of `services` can be found at `http://localhost:3000/api/v1/services`.
 
-The [OpenAPI](https://swagger.io/specification/) documentation can be found at `http://localhost:<port>/<basePath>/<API version>/specs`.
+The [OpenAPI](https://swagger.io/specification/) specification can be found at: 
+
+`http://localhost:<port>/<basePath>/<API version>/specs`
+
+The visual and interactive documentation can be found at:
+
+`http://localhost:<port>/<basePath>/<API version>/docs`
+
+As example, the [visual documentation](http://162.19.74.224/api/v1/docs) of the [Demo collection](https://github.com/OpenTermsArchive/demo-declarations) is publicly available for exploration.
```

Comment:
I get `Cannot GET /api/v1/docs`, please ask for review again when https://github.com/OpenTermsArchive/engine/pull/1015 is deployed 🙂